use_sso_authentication/index.rst

```diff
@@ -0,0 +1,106 @@
+Use SSO authentication locally
+==============================
+
+This chapter describes how to enable the SSO authentication for development purpose.
+
+How SSO works ?
+---------------
+
+| SSO means `Single Sign-On`, their is different implementation/protocols in order to implement a SSO authentication mecanism.
```

Comment:
their is --> there are

Reply:
```suggestion
| SSO means `Single Sign-On`, their are different implementation/protocols in order to implement a SSO authentication mecanism.
```

Reply:
@bgouraud `implementation/protocols` --> `implementations/protocols`